src/main/kotlin/no/nav/syfo/dialogmote/database/DialogmoteDAO.kt

```diff
@@ -135,7 +135,15 @@ class DialogmoteDAO @Inject constructor(
         return namedParameterJdbcTemplate.query(
             query,
             mapSql,
-            dialogmoteRowMapper
+            dialogmoteRowMapper,
+        )
+    }
+
+    fun nullstillDialogmoter(fnr: String) {
```

Comment:
Burde man hatt sånne ikke-prod greier i egen fil / separat fra prodkode?

Reply:
Ja, det høres jo ut som en god idé, men litt usikker på hvordan gjøre det i praksis? Er ikke byggen for dev og prod lik?

Reply:
Jo bygger ikke egne bygg, men tenkte kanskje det ble mer isolert om man hadde en egen DAO for dev-greier. Ikke sikkert det gir mening her, men kan jo ta det i betraktning :D